client/src/pages/dashboard/views/Inbox.svelte

```diff
@@ -56,6 +55,18 @@
     function resetContext() {
         ctx = null;
     }
+
+    $: defer = deferredSnaps.load().catch(err => {
+        assert(err instanceof Error);
+        topToastMessage.enqueue({ title: err.name, body: err.message });
+        return Promise.reject();
+    });
+
+    $: inbox = documentInbox.load().catch(err => {
```

Comment:
Note that this isn't reactive. The `documentInbox` will not react to a `reload` because we have not yet subscribed to the store (typically via the `$` prefix).